terraform/modules/testing/rbac_test_lambda.tf

```diff
@@ -0,0 +1,40 @@
+data "archive_file" "rbac_test_lambda_package" {
+  type        = "zip"
+  source_dir  = "${path.module}/${var.rbac_lambda_file_path}"
+  output_path = "${path.module}/${var.name_prefix}-${var.rbac_lambda_file_path}.zip"
+}
+
+resource "aws_lambda_function" "rbac_test_lambda" {
+  filename         = "${path.module}/${var.name_prefix}-${var.rbac_lambda_file_path}.zip"
```

Comment:
I'd take the output_path of the archive file here, saves constructing it twice.

Reply:
good shout